storage/spider/mysql-test/spider/t/spider_udf_in_where_conds.test

```diff
@@ -0,0 +1,138 @@
+--disable_warnings
+--disable_query_log
+--disable_result_log
+--source test_init.inc
+--enable_result_log
+--enable_query_log
+
+let $CHILD_CREATE_TABLE=
+  CREATE TABLE ta_r (
+    id INT NOT NULL,
+    vc VARCHAR(120),
+    PRIMARY KEY(id)
+  ) $CHILD2_1_ENGINE $CHILD2_1_CHARSET;
+
+let $MASTER_CREATE_TABLE=
+  CREATE TABLE ta_l (
+    id INT NOT NULL,
+    vc VARCHAR(120),
+    PRIMARY KEY(id)
+  ) $MASTER_1_ENGINE $MASTER_1_CHARSET $MASTER_1_COMMENT_2_1;
+
+--echo
+--echo ##### drop and create databases #####
+--connection master_1
+DROP DATABASE IF EXISTS auto_test_local;
+CREATE DATABASE auto_test_local;
+USE auto_test_local;
+if ($USE_CHILD_GROUP2)
+{
+  --connection child2_1
+  DROP DATABASE IF EXISTS auto_test_remote;
+  CREATE DATABASE auto_test_remote;
+  USE auto_test_remote;
+}
+--enable_warnings
+
+--echo
+--echo ##### create table #####
+if ($USE_CHILD_GROUP2)
```

Comment:
What is the intension of branching by the value of `$USE_CHILD_GROUP2`?

Reply:
It seems like a common pattern to me in other Spider tests, i.e. always checking if `$USE_CHILD_GROUP2` before create a table on `$CHILD2_1` or `$CHILD2_2`, even though the value is always TRUE and the test cannot continue on if it's FALSE since Spider will not have any remote tables linked.

Reply:
Thank you for your comment. Yes, it's a common pattern, but it doesn't appear in newer test directories. I'm not quite sure but the patter is possibly outdated one.  You can keep it as is because the test is in the old test directory.